C/other/ternarySearch.c

```diff
@@ -0,0 +1,52 @@
+#include <stdio.h> 
+int ternarySearch(int l, int r, int key, int ar[]) 
+{ 
+    if (r >= l) { 

```

Comment:
Please change the form of the variable name `r`,`l` etc to some good name, that will allow others to view a cleaner code.